admin/client/src/components/grid-field/index.js

```diff
@@ -136,6 +146,7 @@ GridField.propTypes = {
     headerColumns: React.PropTypes.array,
     collectionReadEndpoint: React.PropTypes.object,
     handleDrillDown: React.PropTypes.func,
+    handleEditRecord: React.PropTypes.func,
```

Comment:
This will likely be a temporary prop, since there'll be more actions applicable on a GridField (e.g. publish, unlink, etc) - so we'll need a more powerful registration system. Good enough for now though